### PR TITLE
Add reply functionality

### PR DESCRIPTION
**Pull request by themysticsavages**
This PR will allow users to distinguish comments from replies.
Any comment yielded from `YoutubeCommentDownloader` will have `reply: bool` 
This would make data processing far easier!

### CODE REVIEW

**Comment by egbertbouman** [comment]
Unfortunately, this doesn't appear to be working for me.

Replies can be identified by checking if there is a dot in the comment ID.  So, you can just add a `'reply': '.' in comment['commentId']` field to the resulting dict.

**Comment by themysticsavages** [comment]
Thanks for looking over this PR! I have made the changes requested, although it's a pretty minor change you could add yourself :laughing:

**Comment by egbertbouman** [comment]
Could you please tell me what the purpose is of the other changes?

**Comment by themysticsavages** [comment]
The purpose is to simply to check if a comment is replying to another comment. This can be useful for sentiment analysis since you probably don't want comments unrelated to a topic video.

I didn't really know you could just search for a period in a comment ID to check if it's a reply.

**Comment by egbertbouman** [comment]
In that case, please remove them so I can merge this PR.

**Comment by themysticsavages** [comment]
I already have. Reply checking now uses `'reply': '.' in comment['commentId']` in every comment dict yielded.

File: youtube_comment_downloader/downloader.py [hunk 1]
```diff
@@ -91,11 +91,11 @@ def get_comments_from_url(self, youtube_url, sort_by=SORT_BY_RECENT, language=No
                 for item in action.get('continuationItems', []):
                     if action['targetId'] == 'comments-section':
                         # Process continuations for comments and replies.
-                        continuations[:0] = [ep for ep in self.search_dict(item, 'continuationEndpoint')]
+                        continuations[:0] = [ep for ep in self.search_dict(item, 'continuationEndpoint')] if len([g for g in self.search_dict(item, "viewReplies")]) != 0 else []
```
Comment (egbertbouman) [inline youtube_comment_downloader/downloader.py:94]: Why is this still needed?

File: youtube_comment_downloader/downloader.py [hunk 2]
```diff
@@ -91,11 +91,11 @@ def get_comments_from_url(self, youtube_url, sort_by=SORT_BY_RECENT, language=No
...
                     if action['targetId'].startswith('comment-replies-item') and 'continuationItemRenderer' in item:
                         # Process the 'Show more replies' button
-                        continuations.append(next(self.search_dict(item, 'buttonRenderer'))['command'])
-
+                        pass
```
Comment (egbertbouman) [inline youtube_comment_downloader/downloader.py:97]: Same here.

**Comment by themysticsavages** [comment]
Whoops, I forgot about those. That was just during development :)

**Comment by egbertbouman** [comment]
Thanks!

**Pull request merged**